test/connectors/test-sql-connector.js

```diff
@@ -15,13 +15,66 @@ exports.initialize = function initializeDataSource(dataSource, callback) {
   });
 };
 
-function TestConnector() {
-  SqlConnector.apply(this, [].slice.call(arguments));
+function TestConnector(settings) {
+  SqlConnector.call(this, 'testdb', settings);
   this._tables = {};
 }
 
 util.inherits(TestConnector, SqlConnector);
 
+TestConnector.prototype.escapeName = function(name) {
+  return '`' + name + '`';
+};
+
+TestConnector.prototype.dbName = function(name) {
+  return name.toUpperCase();
+};
+
+TestConnector.prototype.getPlaceholderForValue = function(key) {
+  return '$' + key;
+};
+
+TestConnector.prototype.escapeValue = function(value) {
+  if (typeof value === 'number' || typeof value === 'boolean') {
+    return value;
+  }
+  if (typeof value === 'string') {
+    return "'" + value + "'";
+  }
+  if (value == null) {
+    return 'NULL';
+  }
+  if (typeof value === 'object') {
+    return String(value);
+  }
+  return value;
+};
+
+TestConnector.prototype.toColumnValue = function (prop, val ,escaping) {
```

Comment:
Looks like a type to me, it should be `val, escaping` instead of `val ,escaping`?